### ISSUE TITLE
Why we still do training it the test code?

### ISSUE THREAD

**Issue by HuangBo-Terraloupe** [yhenon/keras-frcnn#105]
In the test code I find 
shared_layer and classifier are set to trainable.
model_rpn.compile(optimizer='sgd', loss='mse')
model_classifier.compile(optimizer='sgd', loss='mse')

Why we still do optimization in the test code?
